api/managementpb/agent/agent.proto

```diff
@@ -140,6 +140,8 @@ message UniversalAgent {
   string version = 37;
   // True if Agent is running and connected to pmm-managed.
   bool is_connected = 38;
+  // True if an exporter agent is exposed on all host addresses.
+  bool expose_exporter_address = 40;
```

Comment:
I wonder if 39 is already reserved?

Reply:
Yeah... it was already been used by `bool comments_parsing_disabled = 39;` - same with the other case too.